### PR TITLE
[PATCH lib] Update relay-mock-network-layer

### PR DESCRIPTION
**Pull request by damassi**
Updates to 2.0 based on https://github.com/1stdibs/relay-mock-network-layer/pull/7.

This allows us to now pass all options over to `graphql-tools` and includes additional validation. See PR above for more details.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -3,9 +3,7 @@
     "@babel/typescript",
     "@babel/react",
     ["@babel/env", {
-      "targets": {
-        "browsers": ["last 2 versions"]
-      },
+      "useBuiltIns": "usage"
```
Comment (damassi) [inline .babelrc]: New way to transpile on demand, similar to `@babel/plugin-transform`

File: package.json [hunk 2]
```diff
@@ -106,6 +106,7 @@
     "cache-loader": "^1.2.2",
     "concurrently": "^3.5.0",
     "conventional-changelog-ember": "^0.3.4",
+    "core-js": "^2.5.7",
```
Comment (damassi) [inline package.json]: peerDep for `react-tracking`

File: package.json [hunk 3]
```diff
@@ -153,7 +154,7 @@
     "webpack-merge": "^4.1.0"
   },
   "dependencies": {
-    "@artsy/palette": "^2.19.1",
+    "@artsy/palette": "^2.19.9",
```
Comment (damassi) [inline package.json]: Clean up some peerDep warnings

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 1eed35bc450edaf962b024331419243519f4a72e

https://deploy-preview-1492--artsy-reaction.netlify.com

**Comment by alloy** [comment]
Great stuff 👍

**Pull request merged**